### PR TITLE
Use helm from kubernetes-tools

### PR DESCRIPTION
**Pull request by sumo-drosiek**

### CODE REVIEW

File: ci/build.sh [hunk 1]
```diff
@@ -93,20 +101,16 @@ function push_helm_chart() {
   set -x
 
   git checkout -- .
-  sudo helm init --client-only
-  sudo helm repo add falcosecurity https://falcosecurity.github.io/charts
-  sudo helm repo add bitnami https://charts.bitnami.com/bitnami
-  sudo helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
```
Comment (pmalek-sumo) [inline ci/build.sh:99]: Why we don't need this here but we ned it below? 🤔  in `generate_overrides`

File: ci/build.sh [hunk 2]
```diff
@@ -161,21 +165,19 @@ function generate_overrides() {
...
+
+  # Get list of templates available for default values.yaml
+  with_files=$(find deploy/helm/sumologic/templates -maxdepth 1 -iname "*.yaml" ! -path "*/otelcol-*" ! -path "*/scc.yaml" ! -path "*/*hpa.yaml" ! -path "*/*psp.yaml" | sed 's#deploy/helm/sumologic/templates#-s templates#g' | sed 's/yaml/yaml /g')
   # shellcheck disable=SC2086
   helm template deploy/helm/sumologic \
     ${with_files} \
     --namespace "\$NAMESPACE" \
-    --name collection \
-    --set dryRun=true >> deploy/kubernetes/fluentd-sumologic.yaml.tmpl \
+    --name-template collection \
```
Comment (pmalek-sumo) [inline ci/build.sh:179]: Why does this parameter change?

File: ci/build.sh [hunk 3]
```diff
@@ -192,13 +194,14 @@ function generate_overrides() {
...
+  # Get list of templates available for default values.yaml
+  with_files=$(find deploy/helm/sumologic/templates/setup -maxdepth 1 -iname "*.yaml" ! -path "*/setup-scc.yaml" | sed 's#deploy/helm/sumologic/templates#-s templates#g' | sed 's/yaml/yaml /g')
   # shellcheck disable=SC2086
   helm template deploy/helm/sumologic \
     ${with_files} \
     --namespace "\$NAMESPACE" \
-    --name collection \
-    --set dryRun=true >> deploy/kubernetes/setup-sumologic.yaml.tmpl \
+    --name-template collection \
+    --set dryRun=true >> deploy/kubernetes/setup-sumologic.yaml.tmpl 2>/dev/null \
```
Comment (pmalek-sumo) [inline ci/build.sh:204]: not specifically related to this PR but how does `>>` work here when used in the middle of parameter list?

File: ci/build.sh (hunk 1 above)
Comment (sumo-drosiek) [reply ci/build.sh:99]: we don't need them below too

File: ci/build.sh (hunk 2 above)
Comment (sumo-drosiek) [reply ci/build.sh:179]: helm2 -> helm3

File: ci/build.sh (hunk 3 above)
Comment (sumo-drosiek) [reply ci/build.sh:204]: same like on the end

**Review by pmalek-sumo — APPROVED** [review]

File: ci/build.sh [hunk 4]
```diff
@@ -192,13 +191,14 @@ function generate_overrides() {
   echo "Generating setup job yaml from helm chart..."
   echo "# This file is auto-generated." > deploy/kubernetes/setup-sumologic.yaml.tmpl
 
-  with_files=$(find deploy/helm/sumologic/templates/setup/ -maxdepth 1 -iname "*.yaml" | sed 's#deploy/helm/sumologic/templates#-x templates#g' | sed 's/yaml/yaml \\/g')
+  # Get list of templates available for default values.yaml
+  with_files=$(find deploy/helm/sumologic/templates/setup -maxdepth 1 -iname "*.yaml" ! -path "*/setup-scc.yaml" | sed 's#deploy/helm/sumologic/templates#-s templates#g' | sed 's/yaml/yaml /g')
```
Comment (perk-sumo) [inline ci/build.sh:195]: `! -path "*/setup-scc.yaml"`?

**Review by perk-sumo — COMMENTED** [review]
Overall 👍  but one thing seems fishy.

File: ci/build.sh (hunk 4 above)
Comment (sumo-drosiek) [reply ci/build.sh:195]: do not take `*/setup-scc.yaml` during generating the `deploy/kubernetes/sumologic.yaml.tmpl`
It is disabled by default in the `values.yaml` so helm3 doesn't see it as template

File: ci/build.sh [hunk 5]
```diff
@@ -14,6 +14,14 @@ err_report() {
 }
 trap 'err_report $LINENO' ERR
 
+function helm() {
+  docker run --rm \
+    -v "$(pwd):/chart" \
+    -w /chart \
+    sumologic/kubernetes-tools:master \
```
Comment (perk-sumo) [inline ci/build.sh:21]: This should not be `master` but rather `latest` or just fixed version.

**Review by perk-sumo — APPROVED** [review]
LGTM but let's make sure we are on fixed `tools` version when it's ready.

**Pull request merged**